Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLIntegration.py

```diff
@@ -152,27 +154,39 @@ def PyInit(self):
         self.setPropertyGroup('MaxQxy', 'I(Qx,Qy) Options')
         self.setPropertyGroup('DeltaQ', 'I(Qx,Qy) Options')
         self.setPropertyGroup('IQxQyLogBinning', 'I(Qx,Qy) Options')
-
-        self.declareProperty(name='BinMaskingCriteria', defaultValue='',
-                             doc='Criteria to mask bins, used for TOF mode,'
-                                 ' for example to discard high and low lambda ranges;'
-                                 'see MaskBinsIf algorithm for details.')
-
         self.declareProperty(WorkspaceGroupProperty('PanelOutputWorkspaces', '',
                                                     direction=Direction.Output,
                                                     optional=PropertyMode.Optional),
                              doc='The name of the output workspace group for detector panels (D33).')
         self.setPropertyGroup('PanelOutputWorkspaces', 'I(Q) Options')
 
+        lambda_range_validator = CompositeValidator()
+        lambda_range_validator.add(FloatArrayOrderedPairsValidator())
+        lambda_range_validator.add(FloatArrayLengthValidator(2))
+        self.declareProperty(FloatArrayProperty('WavelengthRange', [1., 10.], lambda_range_validator),
```

Comment:
Just a nit-pick on consistency, other properties use keywords for setting arguments, while the new property uses only positional arguments.